tests/components/vicare/test_config_flow.py

```diff
@@ -109,7 +111,9 @@ async def test_form_dhcp(hass: HomeAssistant) -> None:
     assert len(mock_setup_entry.mock_calls) == 1
 
 
-async def test_dhcp_single_instance_allowed(hass: HomeAssistant) -> None:
+async def test_dhcp_single_instance_allowed(
+    hass: HomeAssistant, mock_setup_entry: AsyncMock
+) -> None:
```

Comment:
You don't need to check `mock_setup_entry` for aborts.
```suggestion
async def test_dhcp_single_instance_allowed(hass: HomeAssistant) -> None:
```